README.md

```diff
@@ -1,6 +1,6 @@
 # labeledpipe
 
-[![Build Status](<redacted>)](<redacted>)
+[![Build Status](https://travis-ci.org/factset/labeledpipe.svg?branch=master)](https://travis-ci.org/factset/labeledpipe)
 [![Coverage Status](<redacted>)](<redacted>)
```

Comment:
I'm honestly fine either way, but it seem related enough to this PR that you might as well add it.